### PR TITLE
[#943] fix ci

### PR DESCRIPTION
**Pull request by alexander-semenets**
this fixes CI pipelines

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -10,6 +10,7 @@ Prerequisites:
  - GCS Bucket for terraform state with name equal to "<CLUSTER_NAME>-tfstate"
  - SSL certs stored at S3 Bucket (path is configured in vars file)
  - SSH key stored at S3 Bucket (path is configured in vars file)
+ - Reserve Static IP "<CLUSTER_NAME>-nat-gw" and add this IP ad Oauth provider (Keycloak for example)
```
Comment (maxkochubey) [inline README.MD]: `s/IP ad/IP to/`

File: pipelines/ci-integration.Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+                    ]
+                }
+            }
+        }
+    }
+
+    post {
+        always {
+            script {
+                if (env.param_remainTheCluster.toBoolean()) {
```
Comment (mcdoker18) [inline pipelines/ci-integration.Jenkinsfile]: Are you really sure that `param_remainTheCluster` parameter works as expected? And rename it to `param_remain_the_cluster` :D

**Review by mcdoker18 — CHANGES_REQUESTED** [review]

File: pipelines/ci-integration.Jenkinsfile (hunk 2 above)
Comment (alexander-semenets) [reply pipelines/ci-integration.Jenkinsfile]: there is a condition based on the parameter. Else section deletes the cluster.
parameter was renamed as requested.

Comment (alexander-semenets) [reply pipelines/ci-integration.Jenkinsfile]: done

File: README.MD (hunk 1 above)
Comment (alexander-semenets) [reply README.MD]: done

**Review by maxkochubey — APPROVED** [review]

**Review by mcdoker18 — APPROVED** [review]

**Pull request merged**